RickLeinecker/Fortuna
# Prepare 3v3 match api

**Description**
Made an api to prepare the battleRecord for 3v3 battles. Returns the battleRecordId

**Resolves These Issues**
Resolves #365 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1) set a wager for the person who will be challenged
2) set the favoriteTanks to have 3 tanks using MongoDb compass
3) call the api with the personBeingChallengedId and 3 challengerTankIds
4) See if a battleRecord was created accordingly and you receive the battleRecordId

**Final Checklist:**
Go down the list and check them off.
